puppet/modules/koji/templates/mirror_reposync.cron.erb

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Could we use [epp](https://puppet.com/docs/puppet/7/lang_template_epp.html) for new templates? I like the validation it provides and there's also better handling for the `Sensitive` data type. Not that that part is relevant here, but I'd eventually like to convert all templates.